source/include/Date.h

```diff
@@ -54,7 +54,7 @@ class Date {
         Date(Timestamp timestamp, Int32 timeZoneOffset); // this API is conceptually wrong; tzo should be computed from UTC date
         Date(Timestamp timestamp, bool isUTC = false);
         ~Date();
-        static Int32 getLocaletimeZone();
+        static Int32 getLocaletimeZone(Int64 utcTime = 0);
```

Comment:
This is actually different than the isUTC case in other places, because it's really DST rules for the local TZ in both cases.
However, you're right that it isn't needed.  I'll fix the only caller that passes 0, since that should also pass a timestamp.